velox/type/Filter.h

```diff
@@ -1679,6 +1680,8 @@ static inline bool applyFilter(TFilter& filter, T value) {
     return filter.testInt128(value);
   } else if constexpr (std::is_same_v<T, UnscaledShortDecimal>) {
     return filter.testInt64(value.unscaledValue());
+  } else if constexpr (std::is_same_v<T, Date>) {
```

Comment:
Can we add a smoke test to https://github.com/facebookincubator/velox/blob/main/velox/type/tests/FilterTest.cpp?

Reply:
done.